ChangeLog.d/9382.txt

```diff
@@ -0,0 +1,4 @@
+API changes
+   * Align the mbedtls_nist_kw_wrap() and mbedtls_nist_kw_unwrap() functions
+     with the PSA Crypto API. Instead of taking a mbedtls_nist_kw_context as an
+     argument, these functions now take a mbedtls_svc_key_id_t key type.
```

Comment:
I'd remove "type" at the, I think it's potentially confusing, and not helpful: "now take a mbedtls_svc_key_id_t key".